src/main/resources/application.yml

```diff
@@ -0,0 +1,18 @@
+server:
+  port: 8080
+
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/Mentorship
+    username: postgres
+    password: somePassword
```

Comment:
This password should not be exposed

Reply:
I had an issue using env variables here but I have found out a way using the ide to store the environment variables